hack/generate-everything.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+
+run_gen() {
+    echo -e "\n=== $* ===\n"
+    "$@"
+}
+
+# These 3 need to run in this order. Not for any logical reasons, but simply
+# because of the current state of dependence between the generated tasks
+# and the sources they are generated from.
+run_gen hack/build-manifests.sh
+run_gen hack/generate-ta-tasks.sh
+run_gen hack/generate-buildah-remote.sh
+
+run_gen hack/generate-pipelines-readme.py
+
+run_gen hack/update_renovate_json_based_on_codeowners.py -o renovate.json
```

Comment:
I am missing `hack/generate-sast-tasks.sh` in the list of scripts to run.

Reply:
It should be covered by `hack/build-manifests.sh` along with all other Kustomize-generated things

In fact, I think we can remove `hack/generate-sast-tasks.sh` now, WDYT?

Reply:
I have never used `build-manifests.sh` before but it seems to do what is needed.  So yes, I think we can safely remove `generate-sast-tasks.sh`.

Reply:
Removed in latest commit